src/code-generator/test/test-helpers.ts

```diff
@@ -8,6 +9,22 @@ function getRandomFolderPath(basePath: string) {
   return path.join(basePath, randomFolderName);
 }
 
+export async function doesFolderExistInPath(path: string) {
+  return await fsExtra.pathExists(path);
+}
+
+export async function doesFileContainPhrase(filePath: string, phrase: string) {
+  const readStream = createReadStream(filePath, { encoding: "utf-8" });
+  for await (const chunkOfData of readStream) {
+    const chunkAsString = chunkOfData + "";
+    if (chunkAsString.includes(phrase)) {
+      return true;
+    }
+  }
```

Comment:
What if the phrase is split between 2 chunks?

Reply:
Good catch and frankly I also noted this down, handling!